packages/expo/src/scripts/bundle/getAppJson.ts

```diff
@@ -2,6 +2,14 @@ import defaultConfigs from '../../configs';
 import { findFile } from './findFile';
 import path from 'path';
 
+// Transpile files on the fly
+// tslint:disable-next-line:no-var-requires
+require('@babel/register')({
+	extensions: ['.js', '.jsx', '.ts', '.tsx'],
+	presets: ['@bluebase/babel-preset-bluebase'],
```

Comment:
Can we change these to code standards? I'm looking to deprecate this package

Reply:
Related issue #71

Reply:
OK
letting you know of the progress in a while.

Reply:
Actually we have config file as a .ts file which is not yet transpiled. 
The above code is to have this transpiled on the fly when we will be requiring this file.
Can you tell me how will it be replaced by using codestandards repo

Reply:
You just have to change import statement to use this Babel file https://github.com/BlueBaseJS/code-standards/blob/master/babel.config.js

Reply:
But also test this change doesn't break anything

Reply:
OK let me test.